util/cookies.ts

```diff
@@ -1,14 +1,16 @@
 import cookie from 'cookie';
 import Cookies from 'js-cookie';
+import { cookies } from 'next/headers';
 
 // more robust way to get items from cookies without parsing all the time
-export function getParsedCookie(key: string): CookieValue | undefined {
-  const cookieValue = Cookies.get(key);
+export function getCookieByName(name: string): CookieValue | undefined {
+  const cookieValue = cookies().get(name)?.value;
 
   if (!cookieValue) {
     return undefined;
   }
```

Comment:
probably don't need the `if` anymore - function can probably be single line with `return`

Reply:
https://github.com/upleveled/next-js-example-winter-2023-vienna-austria/blob/b185b14ff948fe87a99482d0aa749476c3c23826/util/cookies.ts#L11